### PR TITLE
Added subscription supporting tables.

### PR DESCRIPTION
**Pull request by douradofrazer**

### CODE REVIEW

File: datamodel/initdb.d/05_reference_implementation_support.sql [hunk 1]
```diff
@@ -161,19 +161,16 @@ CREATE TABLE dcsa_im_v3_0.event_subscription (
      transport_call_id varchar(100) NULL,
      signature_method varchar(20) NOT NULL,
      secret bytea NOT NULL,
-     transport_event_type_code varchar(20) ,
      transport_document_reference text NULL,
      transport_document_type_code text NULL,
-     shipment_event_type_code varchar(4) NULL,
```
Comment (nt-as) [inline datamodel/initdb.d/05_reference_implementation_support.sql:167]: This is needed for TNT 2.1. Are you sure it should be removed?

File: datamodel/initdb.d/05_reference_implementation_support.sql [hunk 2]
```diff
@@ -161,19 +161,16 @@ CREATE TABLE dcsa_im_v3_0.event_subscription (
      transport_call_id varchar(100) NULL,
      signature_method varchar(20) NOT NULL,
      secret bytea NOT NULL,
-     transport_event_type_code varchar(20) ,
```
Comment (nt-as) [inline datamodel/initdb.d/05_reference_implementation_support.sql:164]: This is needed for TNT 2.1. Are you sure it should be removed?

File: datamodel/initdb.d/05_reference_implementation_support.sql [hunk 3]
```diff
@@ -184,6 +181,26 @@ CREATE TABLE dcsa_im_v3_0.event_subscription_event_types (
...
+    transport_event_type_code varchar(4) REFERENCES dcsa_im_v3_0.transport_event_type (transport_event_type_code),
+    PRIMARY KEY (subscription_id, transport_event_type_code)
+);
+
+DROP TABLE IF EXISTS dcsa_im_v3_0.event_subscription_equipment_event_type CASCADE;
+CREATE TABLE dcsa_im_v3_0.event_subscription_equipment_event_type (
+    subscription_id uuid NOT NULL REFERENCES dcsa_im_v3_0.event_subscription (subscription_id) ON DELETE CASCADE,
+    equipment_event_type_code varchar(4) REFERENCES dcsa_im_v3_0.equipment_event_type (equipment_event_type_code),
+    PRIMARY KEY (subscription_id, equipment_event_type_code)
+);
```
Comment (HenrikHL) [inline datamodel/initdb.d/05_reference_implementation_support.sql:203]: Don't we need something similar for operations_event_type_code?

Comment (douradofrazer) [reply datamodel/initdb.d/05_reference_implementation_support.sql:203]: yup just added it.

**Review by nt-as — APPROVED** [review]

**Review by nt-as — APPROVED** [review]

**Review by HenrikHL — APPROVED** [review]

**Pull request merged**